go.mod

```diff
@@ -8,17 +8,15 @@ require (
 	github.com/hashicorp/go-hclog v0.12.0 // indirect
 	github.com/hashicorp/go-plugin v1.0.1-0.20190610192547-a1bc61569a26 // indirect
 	github.com/hashicorp/yamux v0.0.0-20181012175058-2f1d1f20f75d // indirect
-	github.com/json-iterator/go v1.1.10 // indirect
 	github.com/kr/pretty v0.2.0 // indirect
 	github.com/kubernetes-csi/external-snapshotter/v2 v2.2.0-rc1
 	github.com/mattn/go-isatty v0.0.12 // indirect
 	github.com/mitchellh/go-testing-interface v1.0.0 // indirect
-	github.com/onsi/gomega v1.8.1 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.4.0
-	github.com/vmware-tanzu/velero v1.4.0
+	github.com/vmware-tanzu/velero v1.5.0
```

Comment:
Since this isn't released yet, won't merging this break the build?

Reply:
No, the `v1.5.0` package is already available. https://pkg.go.dev/github.com/vmware-tanzu/velero@v1.5.0/pkg/restic?tab=doc

Reply:
https://pkg.go.dev/github.com/vmware-tanzu/velero@v1.5.0/pkg/restic?tab=doc#GetPodVolumesUsingRestic

Reply:
😭 

Ok, it works in this case, but that was created mistakenly while testing the release scripts and shouldn't actually exist. I may have to look into deleting that, because it's not going to match the v1.5.0 code that actually gets released. Notice there's no v1.5.0 tag on the Velero repo.

Reply:
Lol. I was actually wondering how the package was available. but didn't think more about it. I am OK if you want to hold this off till 1.5.0 is finally released. Esp considering that it is going to be in the order of days

Reply:
Apparently we can't deleted specific versions, only entire packages.

Hopefully we can overwrite versions on pkg.go.dev :(

Reply:
updated the dependency here to Velero `v1.5.1`